AgentDiscoveries-Backend/src/main/java/org/softwire/training/models/User.java

```diff
@@ -14,11 +15,11 @@
 
     public User() {}
 
-    public User(String username, String hashedPassword, Integer agentId, boolean admin) {
+    public User(String username, String hashedPassword, boolean admin,  int agentId) {
```

Comment:
Similarly, changing the order for testing purposes is a bit odd - might add confusion in others merges that isn't really helpful.

Also, do we definitely want an int rather than an Integer? This gave us headaches before so might not be preferred here!